app/src/main/java/org/p2p/wallet/deeplinks/AppDeeplinksManager.kt

```diff
@@ -58,6 +105,21 @@ class AppDeeplinksManager(
                     isValidScheme && DeeplinkUtils.isValidOnboardingLink(data) -> {
                         rootListener?.triggerOnboardingDeeplink(data)
                     }
+                    isValidScheme && DeeplinkUtils.isValidCommonLink(data) -> {
+                        val screenName = data.host
+                        val target = DeeplinkTarget.fromScreenName(screenName)
+                        target?.let { deeplinkTarget ->
+                            val deeplinkData = DeeplinkData(
+                                target = deeplinkTarget,
+                                pathSegments = data.pathSegments,
+                                args = data.queryParameterNames
+                                    .filter { !data.getQueryParameter(it).isNullOrBlank() }
+                                    .associateWith { data.getQueryParameter(it)!! },
+                                intent = intent
+                            )
```

Comment:
could you move this to a separate function please? 
all other, lgtm

Reply:
I've made the changes in the new commit